### PR TITLE
Remove brackets from source gs-uri for gsutils 

### PR DESCRIPTION
**Pull request by scvannost**

### CODE REVIEW

**Comment by scvannost** [comment]
See https://github.com/CIMAC-CIDC/cidc-schemas/pull/402

File: tests/test_upload.py [hunk 1]
```diff
@@ -286,3 +286,39 @@ def _wait_for_upload(procs, total):
...
+
+
+def test_compose_file_mapping(tmpdir):
+    local_map = {
+        tmpdir.join("local.path"): "test/gcs.1a",
+        tmpdir.join("test.dir"): "test/gcs.1b"
+    }
+    input_map = {
+        "gs://bucket/gcs.path": "test/gcs.2",
+    }
```
Comment (jacoblurye) [inline tests/test_upload.py:298]: Looks like these examples are missing a test for square bracket removal?

File: tests/test_upload.py [hunk 2]
```diff
@@ -286,3 +286,39 @@ def _wait_for_upload(procs, total):
...
+
+    # but must be {str: str}
+    input_map.update({str(k):v for k,v in local_map.items()})
+    input_map["test.dir"] = input_map.pop(str(tmpdir.join("test.dir")))
+
+    upload_job = api.UploadInfo(JOB_ID, JOB_ETAG, GCS_BUCKET, input_map, EXTRA_METADATA, UPLOAD_TOKEN)
+    xlsx = str(tmpdir.join("bar.xlsx"))
+
+    list_of_lists = upload._compose_file_mapping(upload_job, xlsx)
+    output_map = {l[0]:l[1] for l in list_of_lists}
```
Comment (jacoblurye) [inline tests/test_upload.py:312]:
```suggestion
    output_map = dict(upload._compose_file_mapping(upload_job, xlsx))
```

**Review by jacoblurye — APPROVED** [review]

File: tests/test_upload.py (hunk 2 above)
Comment (scvannost) [reply tests/test_upload.py]: in next PR

File: tests/test_upload.py (hunk 1 above)
Comment (scvannost) [reply tests/test_upload.py]: add in next PR

**Pull request merged**